iotedgehubdev/errors.py

```diff
@@ -39,20 +39,16 @@ def __init__(self, msg, ex=None):
         super(EdgeDeploymentError, self).__init__(msg, ex)
 
 
-class ResponseError(Exception):
+class ResponseError(EdgeError):
     def __init__(self, status_code, value):
+        super(ResponseError, self).__init__(value)
```

Comment:
so what is the root cause of failure in 2.7? We need init Exception class?

Reply:
We need call **super** to init base Exception class

Reply:
So do we have ex could be sent to for init?

Reply:
ex is not required, it is used for our only EdgeError, and we do not need it in our case